csatar-plugins/knowledgerepository/updates/builder_table_create_csatar_knowledgerepository_songs.php

```diff
@@ -0,0 +1,36 @@
+<?php namespace Csatar\KnowledgeRepository\Updates;
+
+use Schema;
+use October\Rain\Database\Updates\Migration;
+
+class BuilderTableCreateCsatarKnowledgerepositorySongs extends Migration
+{
+    public function up()
+    {
+        Schema::create('csatar_knowledgerepository_songs', function($table)
+        {
+            $table->engine = 'InnoDB';
+            $table->increments('id')->unsigned();
+            $table->integer('song_type')->nullable();
+            $table->string('title', 200)->nullable();
+            $table->string('author', 200)->nullable();
+            $table->text('text')->nullable();
+            $table->integer('folk_song_type')->nullable();
+            $table->integer('region')->nullable();
+            $table->integer('rhythm')->nullable();
+            $table->string('link', 250)->nullable();
+            $table->string('uploader_csatar_code', 255)->nullable();
+            $table->string('approver_csatar_code', 255)->nullable();
```

Comment:
A cserkesz ID alapjan kell kapcsolodjon ehhez a tablahoz. Az ecset_code nem lehet foreign key. Es majd utana a foreign key-t is allitani a tablara.